src/results/electric_storage.jl

```diff
@@ -43,9 +45,21 @@ function add_electric_storage_results(m::JuMP.AbstractModel, p::REoptInputs, d::
                 ))
             end
          end
+
+        if !isempty(p.techs.electrolyzer)
```

Comment:
For consistency, I think either storage_to_electrolyzer_series_kw and storage_to_compressor_series_kw should be created and set to [] if these tech sets are empty, or they should only be created in the else below if the corresponding sets aren't empty.

Reply:
I think the latter for consistency with elsewhere